app/models/follow.rb

```diff
@@ -27,7 +27,8 @@ class Follow < ApplicationRecord
   scope :follower_tag, ->(id) { where(follower_id: id, followable_type: "ActsAsTaggableOn::Tag") }
 
   counter_culture :follower, column_name: proc { |follow| COUNTER_CULTURE_COLUMN_NAME_BY_TYPE[follow.followable_type] },
-                             column_names: COUNTER_CULTURE_COLUMNS_NAMES
+                             column_names: COUNTER_CULTURE_COLUMNS_NAMES,
+                             touch: true
```

Comment:
I added a touch option in order to update the user's updated_at, which purpose is to update the [user_data cache](https://github.com/forem/forem/blob/master/app/controllers/async_info_controller.rb#L48) in unfollowing a tag. But I'm not sure that updating the user's update_at affects anything else.  if this way is not good, I'll find another way.

Reply:
I think it might be safer to updated `user.last_followed_at` instead than the entire `user.updated_at`, since the cache key you linked uses `user.last_followed_at`.

Reply:
I think this could lead to a lot of unnecessary updates bc of all the callbacks on user. I would rather update the `user_cache_key` in the async controller to maybe include the `@user.cached_followed_tags.count`

When you do this though, be aware that `cached_followed_tags` will hit Redis so ideally we will want to save that to a local variable before creating that async info hash and then use that local variable for the hash and the count for the cache key

Reply:
@Zhao-Andy @mstruve 
Thanks for your comments!

I understood that there was a possibility that updating `user.updated_at` causes a problem when a user follows a tag. So I removed `touch`.

And I considered updating `user.last_followed_at` and adding `@user.cached_followed_tags.count` to `user_cache_key` like you two advised me.
However, I ended up adding `current_user&.following_tags_count` to `user_cache_key` because that needs no additional update or cache and solve the issue.